src/common/SPELLS/paladin.ts

```diff
@@ -285,6 +285,11 @@ const spells = {
     name: 'Blessing of Spring',
     icon: 'ability_ardenweald_paladin_spring',
   },
+  SACRED_WEAPON_TALENT: {
```

Comment:
I'd personally not suffix this with `_TALENT` since its not inside the TALENTS folder, but that's just a matter of taste

Reply:
What would you suffix it with ? I'm not so used to the conventions going around here :)

Reply:
its kind of an odd case because its an ability that you cast that is granted by a talent...but it isn't in the talent data dump. i don't think we really have a convention for that.